packages/react/src/components/Tile/Tile.js

```diff
@@ -295,6 +318,9 @@ SelectableTile.defaultProps = {
   selected: false,
   tabIndex: 0,
   light: false,
+  onClick: () => {},
```

Comment:
`defaultProps` actually might be deprecated for functional components in the future. [The official position of the react team](https://github.com/yannickcr/eslint-plugin-react/issues/2396) is to use ES6 default values for function components.

Regardless, I think it should be one or the other. Splitting the info up between es6 default values and `defaultProps` is confusing so this is fine but maybe worth bringing up in dev meeting.

Reply:
removed the lonely es6 default value for now. Totally hear you that this should be a workstream to drop `defaultProps` in a not too distant future.